ChalzZy/job-spy
# Users can now register and log in

To test, do the following:

1. Run node server `node server.js`
2. Navigate to localhost:3000/register.html
3. Create an account (You should be redirected to the login page)
4. Log in with the same credentials
5. If you successfully log in, you'll be transferred to the main page.
